src/libraries/System.Runtime.Experimental/ref/System.Runtime.Experimental.csproj

```
@@ -17,8 +17,10 @@
     <BuildOutputTargetFolder>ref</BuildOutputTargetFolder>
     <DefineConstants>$(DefineConstants);FEATURE_GENERIC_MATH</DefineConstants>
     <IsPackable>true</IsPackable>
+    <!-- Disable package on release/6.0-maui branch, revert before merging back into release/6.0
     <ServicingVersion>1</ServicingVersion>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    -->
```

revert the branch specific changes:
```suggestion
    <ServicingVersion>1</ServicingVersion>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```